src/main/java/appeng/core/api/definitions/ApiItems.java

```diff
@@ -132,7 +132,8 @@ public ApiItems( final DefinitionConstructor constructor )
 		this.netherQuartzKnife = constructor.registerItemDefinition( new ToolQuartzCuttingKnife( AEFeature.NetherQuartzTools ) );
 
 		this.entropyManipulator = constructor.registerItemDefinition( new ToolEntropyManipulator() );
-		this.wirelessTerminal = constructor.registerItemDefinition( new ToolWirelessTerminal() );
+		IItemDefinition wirelessTerminalbase = constructor.registerItemDefinition( new ToolWirelessTerminal() );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'wirelessTerminalbase' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)